Cargo.toml

```diff
@@ -38,21 +38,20 @@ categories = [
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 reqwest = { version = "0.11.24", features = ["json"] }
+tokio = { version = "1.36.0", features = ["full"] }
```

Comment:
I had to move tokio here to be able to add delays between retries in async workflow